articles/azure-resource-manager/management/lock-resources.md

```diff
@@ -61,6 +61,7 @@ Applying locks can lead to unexpected results. Some operations, which don't seem
 - A read-only lock on a **storage account** protects RBAC assignments scoped for a storage account or a data container (blob container or queue).
 
 - A read-only lock on a **storage account** prevents the creation of a blob container.
+- ++ A read-only lock on a **storage account** prevents the creation of a blob container However,create container operation can be done through both control plane and data plane. And read-only lock only blocks control plane create container request, but user can  still be able create container through data plane (if they have sufficient RBAC permissions and allowed by network). 
```

Comment:
Please address the typos. Suggested:
A read-only lock on a **storage account** prevents the creation of a blob container. However, create operations on a storage account can be done through both the control plane and data plane. Read-only locks only block control plane create requests, but a user can still perform a valid create operation on the resource through the data plane.

Reply:
I have changed as pe your suggestions